images/kubevirt-infra-bootstrap/Dockerfile

```diff
@@ -16,3 +16,14 @@ RUN apt-get update \
         python3 \
         python3-pip \
         && rm -rf /var/lib/apt/lists/*
+RUN export GOROOT=/tmp/go/root \
```

Comment:
We better use some standard directory. I guess /opt would fit. There are no guarantees about /tmp surviving